linux_os/guide/system/bootloader-zipl/zipl_enable_selinux/rule.yml

```diff
@@ -0,0 +1,31 @@
+documentation_complete: true
+
+prodtype: rhel8
+
+title: 'Ensure SELinux Not Disabled in zIPL'
+
+description: |-
+    To ensure SELinux is not disabled at boot time,
+    check that no boot entry in <tt>/boot/loader/entries/*.conf</tt> have <tt>selinux=0</tt>
+    included in its options.
+    Make sure <tt>/etc/zipl.conf</tt> doesn't contain <pre>image = </pre> setting,
+    as {{{ full_name }}} adheres to Boot Loader Specification (BLS).
+    And run <pre>zipl</pre> command so that <tt>/boot/bootmap</tt> is updated.
+
+rationale: |-
+    Disabling a major host protection feature, such as SELinux, at boot time prevents
+    it from confining system services at boot time.  Further, it increases
+    the chances that it will remain off during system operation.
+
+severity: medium
+
+ocil_clause: 'SELinux is disabled at boot time'
+
+ocil: |-
+    Inspect <tt>/etc/default/grub</tt> for any instances of <tt>selinux=0</tt>
```

Comment:
Does it make sense to have this `grub` thing?

Reply:
No, :joy: